core/services/major_tom/requirements.txt

```diff
@@ -0,0 +1,6 @@
+psutil==5.9.5
+requests==2.31.0
+speedtest-cli==2.1.3
+Flask==2.3.2
```

Comment:
We should use setup.py. How is that being installed ? We do not install `requeriments.txt` do we ?